requirements.txt

```diff
@@ -7,7 +7,7 @@ jsonpatch==1.32
 jsonschema==4.4.0
 logging-color-formatter==1.0.2
 newrelic==7.2.4.171
-psycopg2-binary==2.9.3
+psycopg2==2.9.3
```

Comment:
This change is because `kinto[postgresql]` requires `psycopg2` (not `psycopg2-binary`)

Reply:
psycopg2-binary is psycopg2 precompiled, we shoul keep the binary version IMHO.

Reply:
> psycopg2-binary is psycopg2 precompiled, we should keep the binary version IMHO.

I would prefer that as well, but I think in order to do this, we'll have to change `psycopg2` to `psycopg2-binary` in [`setup.cfg`](https://github.com/Kinto/kinto/blob/a9e46e57de8f33c7be098c6f583de18df03b2824/setup.cfg#L75). Would that be acceptable? I didn't know if there was a good reason we needed to build `psycopg2` when installing `kinto`.

Reply:
I think it makes sense to align the dependencies names. Shall we try `psycopg2-binary` everywhere?

Reply:
> Shall we try `psycopg2-binary` everywhere?

Yep, we certainly can. I was just ensuring there wasn't a good reason to build `psycopg2`. From [their docs](https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary):

> If you are the maintainer of a published package depending on psycopg2 you shouldn’t use psycopg2-binary as a module dependency. *For production use you are advised to use the source distribution.*

and 

> *Warning*

> The psycopg2 wheel package comes packaged, among the others, with its own libssl binary. This may create conflicts with other extension modules binding with libssl as well, for instance with the Python ssl module: in some cases, under concurrency, the interaction between the two libraries may result in a segfault. In case of doubts you are advised to use a package built from source.

Reply:
@Natim I'm going to merge this PR as-is, then open a follow-up issues where we can further discuss using `psycopg2` vs `psycopg2-binary`